charts/linkerd-crds/templates/policy/httproute.yaml

```diff
@@ -16,6 +16,7 @@ spec:
     listKind: HTTPRouteList
     plural: httproutes
     singular: httproute
+    shortNames: []
```

Comment:
I'd love to fill this field as well as for example ArgoCD will constantly try to sync this CRD.

Reply:
@michalschott, if `shortNames` is completely omitted rather than an empty array, all will be good?